src/UWPClientLibrary/Microsoft.FactoryOrchestrator.UWPClient.csproj

```diff
@@ -23,7 +22,7 @@
     <TargetName>Microsoft.FactoryOrchestrator.UWPClient</TargetName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.17763.0</TargetPlatformVersion>
+    <TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.19041.0</TargetPlatformVersion>
     <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>

```

Comment:
Should we update the minimum version  ?

Reply:
WinUI 3 highly encourages updating the target platform version to build 19041. As for the minimum version, they recommend build 17763 which is what we already have